src/GnuTLS.jl

```diff
@@ -8,16 +8,33 @@ include("../deps/deps.jl")
 
 import Base: isopen, write, read, readall, readavailable, close, show, nb_available, eof
 
-
+const GNUTLS_MINSECURE_VER = @compat Dict(
+	v"2.12"	=>	23,
+	v"3.1"	=>	28,
+	v"3.2"	=>	20,
+	v"3.3"	=>	10
+)
+
+const WRITE_RETURN_TYPE = VERSION >= v"0.4-" ? UInt64 : Int64
```

Comment:
I was scared to do this since `UInt64` is nothing on 0.3.  We'll see what happens, I suppose!  :D